packages/smooth_app/lib/pages/product/product_page.dart

```diff
@@ -201,9 +94,9 @@ class _ProductPageState extends State<ProductPage> {
               ),
               label: appLocalizations.label_preferences,
             ),
-            BottomNavigationBarItem(
-              icon: const Icon(Icons.playlist_add),
-              label: appLocalizations.label_lists,
+            const BottomNavigationBarItem(
+              icon: Icon(Icons.copy),
+              label: 'Copy',
```

Comment:
I think "add to list" still fits better here

Reply:
Not really:
* we don't have enough space for that - it's a bottom navigation bar item
* there's still ambiguity with product list, shopping list and pantries
* anyway, I'll let the translators worry about that :)